packages/plugins/src/index.ts

```diff
@@ -44,7 +44,7 @@ import * as compound from './plugins/compound';
 import * as stargate from './plugins/stargate';
 import * as stader from './plugins/stader';
 import * as uniswap from './plugins/uniswap';
-import * as balancer from './plugins/balancer';
+import * as balancer from './plugins/balancer2';
```

Comment:
Why did we add a new folder (and deleted the old one) vs simply updating the code that already existed ?

If there were significant changes between `balancer` and `balancer2`, why not do:
```suggestion
import * as balancer2 from './plugins/balancer2';
```